src/arviz_plots/plots/compare_plot.py

```diff
@@ -27,44 +35,62 @@ def plot_compare(
 
+    The ELPD can only be interpreted in relative terms. But differences in ELPD less than 4
+    are considered negligible [3]_.
+
     Parameters
     ----------
     comp_df : pandas.DataFrame
-        Result of the :func:`arviz_stats.compare` method.
-    similar_shade : bool, optional
-        If True, a shade is drawn to indicate models with similar
-        predictive performance to the best model. Following [3]_,
-        models are considered similar if the difference in ELPD is
-        less than 4. Defaults to True.
+        Usually this will be the result of the :func:`arviz_stats.compare` function.
+        It is assumed that the DataFrame has two columns one named `elpd`, `mlpd`, or `gmpd`,
+        the other named `se`, and the index is the model names. Additionally,
+        it is assumed that the first row of the DataFrame is the top model.
     relative_scale : bool, optional.
-        If True scale the ELPD values relative to the best model.
+        If True, the `stats` values are scaled relative to the best model.
+        Defaults to True.
+    rotated : bool, optional
+        If True, the plot is rotated, with models on the y-axis and ELPD on the x-axis.
         Defaults to False.
+    hide_top_model : bool, optional
+        If True, the top model (first row of `comp_df`) will not appear as a point with error bars
+        or in the axis labels. Its performance can still be accessed by the visuals `ref_line`
+        and/or `ref_band`. Defaults to False.
     backend : {"bokeh", "matplotlib", "plotly"}
         Select plotting backend. Defaults to rcParams["plot.backend"].
-    figsize : tuple of (float, float), optional
-        If `None`, size is (10, num of models) inches.
     visuals : mapping of {str : mapping or bool}, optional
         Valid keys are:
 
         * point_estimate -> passed to :func:`~arviz_plots.backend.none.scatter`
         * error_bar -> passed to :func:`~arviz_plots.backend.none.line`
-        * ref_line -> passed to :func:`~arviz_plots.backend.none.line`
-        * shade -> passed to :func:`~arviz_plots.backend.none.fill_between_y`
-        * labels -> passed to :func:`~arviz_plots.backend.none.xticks`
-          and :func:`~arviz_plots.backend.none.yticks`
+        * ref_line -> passed to :func:`~arviz_plots.backend.none.hline` or
+        :func:`~arviz_plots.backend.none.vline` depending on the
+        ``rotated`` parameter.
+        * ref_band -> passed to :func:`~arviz_plots.backend.none.hspan` or
+        :func:`~arviz_plots.backend.none.vspan` depending on the
+        ``rotated`` parameter. Defaults to ``False``.
+        * similar_line -> passed to :func:`~arviz_plots.backend.none.hline` or
+        :func:`~arviz_plots.backend.none.vline` depending on the
+        ``rotated`` parameter. Defaults to ``False``.
+        * labels -> passed to :func:`~arviz_plots.backend.none.xticks` and
+        :func:`~arviz_plots.backend.none.yticks`
```

Comment:
multiline bullet points need to be indented, otherwise sphinx thinks the list ends there and we are starting a new paragraph (but sends a warning which caught it: https://app.readthedocs.org/projects/arviz-plots/builds/29276643/#283097490--347)

```suggestion
        * ref_line -> passed to :func:`~arviz_plots.backend.none.hline` or
          :func:`~arviz_plots.backend.none.vline` depending on the
          ``rotated`` parameter.
        * ref_band -> passed to :func:`~arviz_plots.backend.none.hspan` or
          :func:`~arviz_plots.backend.none.vspan` depending on the
          ``rotated`` parameter. Defaults to ``False``.
        * similar_line -> passed to :func:`~arviz_plots.backend.none.hline` or
          :func:`~arviz_plots.backend.none.vline` depending on the
          ``rotated`` parameter. Defaults to ``False``.
        * labels -> passed to :func:`~arviz_plots.backend.none.xticks` and
          :func:`~arviz_plots.backend.none.yticks`
```